### PR TITLE
Add explanation to virtualenv and pycharm setup

### PR DESCRIPTION
**Pull request by artnie**
Just documentation on virtualenv

### CODE REVIEW

File: doc/source/installation.rst [hunk 1]
```diff
@@ -160,6 +161,12 @@ After installing pandoc, install sphinx on your device.
 
     sudo apt install python3-sphinx
 
+Install IPython
+
+.. code-block:: shell
+
+    sudo pip3 install ipython
+
```
Comment (Tigul) [inline doc/source/installation.rst:169]: Why is ipython relevant for building the documentation ?

Comment (tomsch420) [reply doc/source/installation.rst:169]: Ipython is irrelevant for pycram.

Comment (Tigul) [reply doc/source/installation.rst:169]: @artnie Could you remove the codeblock from the documentation

Comment (artnie) [reply doc/source/installation.rst:169]: Ah I see, it's just warnings you get from building the ipynb files when ipython is not installed.

Comment (tomsch420) [reply doc/source/installation.rst:164]: with building the ipynb files you mean building the doc?

**Review by tomsch420 — APPROVED** [review]

**Pull request merged**